provisioning_virtual_machines_and_hosts/_topics/catalogs_and_services.md

```diff
@@ -372,6 +372,125 @@ Service methods have been split based on purpose.
 
 7.  Click **Save** or **Add** if you are adding this to a new instance.
 
+### Provisioning Dialog
+
+The following example shows how to create a Provisioning Dialog that modifies the parameters for provisioning a small, medium, or large service.
+
+The `provision-vm-small-med-large.asl` workflow runs for every provisioned VM and is responsible for modifying the VM size. This workflow is the `<ManageIQ>` service Provisioning Entry Point.
+
+You need to set the ManageIQ Provisioning service entry point to workflow script. For example, this workflow is run for every provisioned Virtual Machine (VM), and modifies the VM size.
+
+Use the following sections and follow them in order to provision a VM service, create Provisioning Dialog and modify the paramaters.
+
+#### Create a workflow (optional)
+
+**Note**: If you want to use the existing `provision-vm-small-med-large.asl` workflow script without any changes, you can skip this section and proceed to `Add a Workflow Repository` section.
+
+1. Fork `https://github.com/ManageIQ/workflows-examples` repository or create your own repository.
+2. Clone the repo locally by using the clone command:
+
+   ```bash
+   git clone https://github.com/<owner>/workflows-examples
+   ```
+   Where `<owner>` is the owner of the repository.
+
+3. Create a new branch:
+
+   ```bash
+   git checkout -b sm-md-lg
+   ```
+3. Create your own script or edit the existing script that is provided. The script used in this example is `provision-vm-small-med-large.asl`.
+4. Push code changes to your branch.
+
+#### Add a Workflow Repository
+
+Use the following steps to add a Workflow Repository.
+
+1. Click **Automation** > **Embedded Workflows** > **Repositories**.
+2. Click **Configuration** > **Add new Repository**.
+3. Provide the **Name** and **URL** for the repository:
+   - **Name**: Local name to identify this repository. For example, `Example Workflows`.
+   - **Url**: Git repository URL. For example, `https://github.com/ManageIQ/workflows-examples`.
+   - **SCM Branch**: Git repository branch name. For example, leave this field blank if you want to use the existing `provision-vm-small-med-large.asl` script blank or use a branch name like `sm-md-lg` if you have edited or created your own workflow script.
```

Comment:
Something feels wrong with this sentence - feels like a copy/paste error (not sure)?

```suggestion
   - **SCM Branch**: Git repository branch name. For example, leave this field blank if you want to use the existing `provision-vm-small-med-large.asl` script from the master branch, or use a branch name like `sm-md-lg` if you have edited or created your own workflow script.
```